artillery-tests/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "@polkadot/api-contract": "^10.6.1",
+    "@polkadot/keyring": "^12.1.2"
+  }
+}
```

Comment:
```suggestion
{
  "scripts": {
    "artillery": "artillery run --output report.json tests/substrate.yml",
    "artillery-report": "artillery report report.json"
  },
  "dependencies": {
    "@polkadot/api-contract": "^10.6.1",
    "@polkadot/keyring": "^12.1.2",
    "artillery": "^2.0.0-32",
    "artillery-engine-substrate": "akorchyn/artillery-engine-substrate#42dfcf009e"
  }
}
```

Reply:
Did you try uninstalling artillery globally?  This is working for me on my Pop!_OS box.

Reply:
yeah probably that was the issue